### PR TITLE
Replace Staking with Protocol

### PR DESCRIPTION
**Pull request by frm**

### CODE REVIEW

File: src/popup/components/common/NavBar/index.tsx [hunk 1]
```diff
@@ -88,18 +48,14 @@ const RootContainer = styled.div`
...
   const history = useHistory();
 
-  const stakingDetails: any = useUserSelector(getStakingDetails);
   const credentials = useUserSelector(getAttestedClaims);
 
   const onClickExport = async () =>
     exportFile(credentials.serialize(), "fractal_wallet.backup");
-  const onClickImport = () => windows.openTab("upload.html");
-  const onClickReconnect = () => history.push(RoutesPaths.CONNECT_WALLET);
+  const onClickImport = () => WindowsService.openTab("upload.html");
```
Comment (djcouto) [inline src/popup/components/common/NavBar/index.tsx:58]: This two lines, and corresponding methods, must be deleted

File: src/types/plugin/Common.ts [hunk 2]
```diff
@@ -16,8 +16,8 @@ export interface ICollection<T> extends ISerializable, Array<T> {
   removeByField: (field: string, value: any) => void;
 }
 
-export interface SignedNonce {
-  signer: string;
-  nonce: string;
-  signature: string;
-}
+// export interface SignedNonce {
```
Comment (djcouto) [inline src/types/plugin/Common.ts]: Delete this

**Review by djcouto — CHANGES_REQUESTED** [review]

File: src/popup/components/common/NavBar/index.tsx (hunk 1 above)
Comment (frm) [reply src/popup/components/common/NavBar/index.tsx:58]: They're deleted in #64

File: src/types/plugin/Common.ts (hunk 2 above)
Comment (frm) [reply src/types/plugin/Common.ts]: 👍

**Review by djcouto — APPROVED** [review]

**Pull request merged**